salt/utils/schedule.py

```diff
@@ -408,23 +408,27 @@ def run_job(self, name):
 
         # Grab run, assume True
         run = data.get('run', True)
+        run_jobs_in_background = self.opts.get('run_jobs_in_background', True)
```

Comment:
should it be `run_schedule_jobs_in_background`? On the salt side, `opts` is the **whole** set of options....

Reply:
Should probably be `run_schedule_jobs_in_background`.

Reply:
Updated